org.opent2t.sample.hub.superpopular/com.wink.hub/js/thingTranslator.js

```diff
@@ -88,6 +88,152 @@ class Translator {
         // Make the async request
         return this._makeRequest(requestPath, 'PUT', putPayloadString);
     }
+    
+    /**
+     * Subscribes to a Wink pubsubhubbub feed.  This function is designed to be called twice.
+     * The first call will contain just the callbackUrl, which will be subscribed for postbacks
+     * from the Wink cloud service.
+     * The second call will contain the request, and response for a GET to the callbackUrl that was
+     * provided in order to complete the verification of the subscription.
+     * 
+     * The expected sequence is as follows:
+     * - Call subscribe with a callback url, get a response.
+     * - Http server running at the callback url receives a GET request, this request and a response string
+     *      are passed to a 2nd call to subscribe() (callbackUrl is ignored in this case)
+     * - Http server responds to the GET with the verificationResponseContent that was populated by subscribe
+     * - Future POST calls to the callback url are passed to the device translator get*Uri function
+     * 
+     * All calls to subscribe will return the expiration time of the subscription.  For Wink, this is
+     * currently 24 hours.  Subsequent calls to subscribe will refresh the expiration time, and in the
+     * case of Wink/Pubsubhubbub will not require verification.
+     * 
+     * @param {string} deviceType - Device Type (e.g. 'thermostats')
+     * @param {string|number} deviceId - Id for the specific device
+     * @param {string} callbackUrl - Callback url for feed postbacks
+     * @param {HttpRequest} verificationRequest - GET request received by the server at a previously subscribed
```

Comment:
>HttpRequest [](start = 15, length = 11)

Is this the entire HTTPREquest? Or are we interested in only the Headers/Parameters specified in the GET request? passing the entire request object seems like overkill?